src/compiler.ts

```diff
@@ -462,6 +462,36 @@ export class Compiler extends DiagnosticEmitter {
     return module;
   }
 
+  addNullPointerExceptionHandling(module: Module) {
+    if (this.options.hasFeature(Feature.EXCEPTION_HANDLING)) {
+      let message = this.ensureStaticString("null is not a function");
```

Comment:
This returns an index that is emitted but for some reason the string isn't interned in a data section AFAICT. Anyone see what I did wrong?

Also, bikeshedding welcome on the error message, as always.